core/src/test/scala/org/apache/spark/util/AkkaUtilsSuite.scala

```diff
@@ -61,9 +55,14 @@ class AkkaUtilsSuite extends SparkFunSuite with LocalSparkContext with ResetSyst
 
     val slaveRpcEnv = RpcEnv.create("spark-slave", hostname, 0, conf, securityManagerBad)
     val slaveTracker = new MapOutputTrackerWorker(conf)
-    intercept[akka.actor.ActorNotFound] {
+    try {
       slaveTracker.trackerEndpoint =
-        slaveRpcEnv.setupEndpointRef("spark", rpcEnv.address, MapOutputTracker.ENDPOINT_NAME)
+        slaveRpcEnv.setupEndpointRef(rpcEnv.address, MapOutputTracker.ENDPOINT_NAME)
+    } catch {
+      case e: RuntimeException =>
+        assert(e.getMessage.contains("javax.security.sasl.SaslException"))
+      case e: SparkException =>
+        assert(e.getMessage.contains("Message is dropped because Outbox is stopped"))
```

Comment:
Actually I think that most of these tests are now redundant; the transport library's auth code is already well covered in the network-common tests.